frontend/src/metabase/collections/components/PinnedItemOverview/PinnedItemOverview.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import _ from "underscore";
+import { t } from "ttag";
+
+import PinnedItemCard from "metabase/collections/components/PinnedItemCard/PinnedItemCard";
+import { Item, Collection } from "metabase/collections/utils";
+
+import { Container, Grid, SectionHeader } from "./PinnedItemOverview.styled";
+
+type Props = {
+  items: Item[];
+  collection: Collection;
+  onCopy: (items: Item[]) => void;
+  onMove: (items: Item[]) => void;
+};
+
+function PinnedItemOverview({ items, collection, onCopy, onMove }: Props) {
+  const sortedItems = _.sortBy(items, item => item.name);
+  const {
+    card: cardItems = [],
+    dashboard: dashboardItems = [],
+    dataset: datasetItems = [],
+  } = _.groupBy(sortedItems, "model");
+
+  return items.length ? (
+    <Container data-testid="pinned-items">
+      {cardItems.length > 0 && (
+        <Grid>
+          {cardItems.map(item => (
+            <PinnedItemCard
+              key={item.id}
+              item={item}
+              collection={collection}
+              onCopy={onCopy}
+              onMove={onMove}
+            />
+          ))}
+        </Grid>
+      )}
+      {dashboardItems.length > 0 && (
+        <Grid>
+          {dashboardItems.map(item => (
+            <PinnedItemCard
+              key={item.id}
+              item={item}
+              collection={collection}
+              onCopy={onCopy}
+              onMove={onMove}
+            />
+          ))}
+        </Grid>
+      )}
+      {datasetItems.length > 0 && (
+        <div>
+          <SectionHeader>
+            <h4>{t`Datasets`}</h4>
```

Comment:
Sorry for the confusion, but we decided to call them "Models" instead of "Datasets". It probably isn't mirrored in products docs or Figma right now as they were prepared before the decision to go with "Models". Could you please fix that and use "models" in the future? 🙏 

Also, we tend to use `dataModels` in code (meaning variables / methods / etc, not user-facing strings) to avoid ambiguity and confusion with e.g. search `models` param.

Reply:
That's my bad that the mock text didn't get updated. 🤦🏼‍♂️

Reply:
good catch; fixed. collection items still have a `model` property of `"dataset"` so we'll need to change that eventually as well